src/bundle/Resources/public/scss/_notifications.scss

```diff
@@ -3,6 +3,7 @@
     bottom: 0;
     width: 100%;
     line-height: 2.25;
+    z-index: 1;
```

Comment:
@dew326 is the value big enough? 🙂

Reply:
@micszo it's big enough.

Reply:
It will work, but for me, it should be something bigger like 50000 (UDW has 99999), just for safety.

Reply:
And what if someone wants to create an element that is before all other but behind the notifications. With z-index 1 he has no space for adding rule to own element

Reply:
@dew326 ok, safety first. I will change to 50000